studio/schemas/post.ts

```diff
@@ -18,12 +29,14 @@ export default defineType({
         source: 'title',
         maxLength: 96,
       },
+      group: 'config',
```

Comment:
🔧 Can we add the following to make `slug` a required field?:
```suggestion
      validation: Rule => Rule.required(),
      group: 'config',
```

Reply:
Sorry, you did ask for this!